### PR TITLE
Added Czech translation

### PR DESCRIPTION
**Pull request by tirnovar**
Hi, according to the e-mail agreement, I have prepared a Czech language version for Bravo. I didn't change what is marked as "Do not translate," including"locate" in the code. I hope it's all right. I'll adjust it :)

### CODE REVIEW

File: src/wwwroot/src/model/i18n/cz.ts [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+ * Bravo for Power BI
+ * Copyright (c) SQLBI corp. - All rights reserved.
+ * https://www.sqlbi.com
+*/
+import { Locale } from '../i18n';
+import { strings as _ } from '../strings';
+
+const locale: Locale = {
+
+    locale: "en", //DO NOT TRANSLATE
```
Comment (albertospelta) [inline src/wwwroot/src/model/i18n/cz.ts:11]: Should be `locale: "cz"`

File: src/wwwroot/src/model/i18n/cz.ts [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+    name: "English", //DO NOT TRANSLATE
```
Comment (albertospelta) [inline src/wwwroot/src/model/i18n/cz.ts:12]: Should be `name: "Czech"`
Also, can you please remove `//DO NOT TRANSLATE` as this is a misleading comment from the original source file.

**Review by albertospelta — CHANGES_REQUESTED** [review]

**Review by albertospelta — CHANGES_REQUESTED** [review]
Could you move the file from folder `src/wwwroot/src/model/i18n/` to folder `src/Scripts/model/i18n/`? This is required because we have recently changed the path to these files in the repository. Sorry about that!

**Comment by tirnovar** [comment]
I hope that I have fixed it all.

**Pull request merged**